content/pages/purchasing.rst

```diff
@@ -6,7 +6,7 @@ EME 185 A/B Capstone MAE Purchasing And Out-Of-Pocket Reimbursement Guidelines
 Key Guidelines
 ==============
 
-- Expenses must first be approved before purchased.
+- Expenses must first be approved before purchased.  There is a cap of two additional purchase requests.
```

Comment:
Need to be careful about difference in BOM approvals and purchase requests.

If they are doing purchasing through OPS they can submit as many of them as they want.

If the TAs are doing purchasing through OPS then they can 1 one per two weeks.

Both of those are different than their 3 BOM submissions (orginal + 2 change orders).